docs/questions/native-editor/sql-parameters.md

```diff
@@ -324,16 +221,125 @@ WHERE
 {% endraw %}
 ```
 
-### SQL syntax
+## Customizing dropdown lists and search box values
+
+With Text and Field filter variables, you can tell Metabase what values people can choose from when using a filter with a dropdown list or search box.
 
-Make sure your SQL dialect matches the database you've selected. Common errors:
+1. In the native editor, add a {% raw %}{{variable}}{% endraw %} in double braces.
+2. If the sidebar doesn't open, you can click on the **{x}** icon on the right to open the **Variables** sidebar.
+3. In the **Settings** tab, set the **Variable type** to either "Text" or "Field Filter".
+4. Scroll down to **How should users filter on this variable?** Pick either **Dropdown list** or **Search box**.
```

Comment:
I think the " Scroll down" doesn't fit here, as the sidebar options are fully visible without the need to scroll

<img width="1508" alt="Screenshot 2024-03-11 at 12 09 52 PM" src="https://github.com/metabase/metabase/assets/154609371/fec92031-f432-4321-8cfd-1b6731795a45">